logstash-configs/conf.d/00-input-elasticsearch-1w.conf

```diff
@@ -1,10 +0,0 @@
-input {
```

Comment:
why are you touching this file?

Reply:
I delete this file cuz the change has been incorporated into file "logstash-configs-prod/conf.d/00-input-elasticsearch.conf"